float32 z
---
float32 x
float32 axis_d
float32 axis_c